fundingmanager.go

```diff
@@ -53,6 +53,10 @@ const (
 	// for the funding transaction to be confirmed before forgetting about
 	// the channel. 288 blocks is ~48 hrs
 	maxWaitNumBlocksFundingConf = 288
+
+	// reservationTimeout is the length of idle time that must pass before
+	// a reservation is cancelled
+	reservationTimeout = 1 * time.Minute
```

Comment:
1) If you make this part of the `fundingConfig` then we can set it to a small value during tests.
2) 1 minute is a bit small maybe? I was thinking this should be on the order of ~10 minutes, but don't really have a convincing reason other than 1 minute being close to what we can expect for network latency+processing time on a slow network/computer.

Reply:
Done.